ssl.go

```diff
@@ -114,7 +116,7 @@ func findCertRemainingValidity(certChains [][]*x509.Certificate) (float64, *x509
 
 	// find chain with max remaining validity
 	for _, chain := range certChains {
-		fmt.Printf("checking chain of len %d", len(chain))
+		// fmt.Printf("checking chain of len %d", len(chain))
```

Comment:
The line probably should be un-commented or removed

Reply:
It needs to be removed then, because it breaks JSON output. Fixed.